pkgs/servers/gpsd/default.nix

```diff
@@ -44,9 +45,16 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     patchShebangs .
     sed -e "s|systemd_dir = .*|systemd_dir = '$out/lib/systemd/system'|" -i SConstruct
-    scons prefix="$out" leapfetch=no gpsd_user=${gpsdUser} gpsd_group=${gpsdGroup} \
-        systemd=yes udevdir="$out/lib/udev" \
-        python_libdir="$out/lib/${python2Packages.python.libPrefix}/site-packages"
+    export CFLAGS="-I${pps-tools.dev}/include"
```

Comment:
Add `pps-tools` to `buildInputs` instead.

Reply:
Doesn't seem to be detected in that case.